homeassistant/components/zwave/config_flow.py

```diff
@@ -12,6 +13,22 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+VENDOR_IDS = (('Aeon Labs Gen5', 'VID_0658'),)
+
+
+def _get_z_stick():
+    import serial.tools.list_ports
+
+    for port in serial.tools.list_ports.comports(include_links=False):
+        if port.vid is None:
+            continue
+        for vendor, vid in VENDOR_IDS:
+            if vid == 'VID_' + hex(port.vid)[2:].zfill(4):
+                if sys.platform.startswith('win'):
+                    return vendor + ' : ' + '\\\\.\\' + port.device
+                else:
+                    return vendor + ' : ' + port.device
```

Comment:
So, I think there's a problem with this approach of including the vendor name. For an Aeotec stick (using Linux paths as an example), this might result in something like `Aeon Labs Gen5 : /dev/ttyACM0`, right?

Well, the VID 0x0658 refers to Sigma Labs, not Aeon Labs/Aeotec. Other USB sticks that use the Sigma Labs (now Silicon Labs) Z-Wave SOC report the same VID. Besides the Aeotec I also have a Silicon Labs UZB3 that reports 0658. I think the UZB1 also has the same VID. This is going to be confusing for people not using the Aeotec Z-Stick. I don't really know how many USB sticks out there use the Sigma design, but it's at least 3.

If you wanted to also use the PID (product id), you can't really. The Aeotec reports a PID of 0x200. So does the UZB3. Who knows how many other sticks out there also report 0658:0200.

In the end, it's not going to be possible to accurately describe every USB stick by name, and calling all SIgma-based sticks "Aeon Labs/Aeotec" is not right. I would recommend just annotating the path name with some generic indicator, instead of a device name, e.g. `Z-Wave USB : <usb_path>` or something, or just leave off the description entirely and still provide the suggestion.